services/nftStorage.ts

```diff
@@ -49,13 +50,16 @@ export const getKey = async (address: string): Promise<PinningKey> => {
   return { expiry, token }
 }
 
+export const PinFileToIPFSwithRetries = (file: Blob) =>
```

Comment:
```suggestion
export const PinFileToIPFSwithRetries = (file: Blob) =>
```

// please variables are named lowercase

Reply:
done